### PR TITLE
Infrastructure WG Roadmap

### PR DESCRIPTION
**Pull request by daviddias**
Preview: https://github.com/ipfs/roadmap/blob/roadmap-wg-infrastructure/WG_INFRASTRUCTURE.md

### CODE REVIEW

File: WG_INFRASTRUCTURE.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* Manage and maintain the IPFS Bootstrapper nodes.
+* Manage and maintain the IPFS Preload nodes
+* Manage and maintain the IPFS pinning service, pinbot.
+* Monitor services and hosts used by the IPFS dev teams.
+* Provide wisdom to other users that want to host IPFS nodes.
+
+## Current Projects
+* Gateway maintenance/redeployment
+* js preload/bootstrapper redeployment
+* CI prototyping for js-ipfs
```
Comment (momack2) [inline WG_INFRASTRUCTURE.md:15]: CI support in general seems like a focus area for infra in improving project velocity and contributor experience - @eefahy are there any additional milestones about developer support that we should put on the ipfs-infra roadmap?

Comment (eefahy) [reply WG_INFRASTRUCTURE.md:15]: One of the PL level infra goals is: "Provide easy to use interfaces for project and PL needs" which is a bit of a catch all for developer support tooling. Is something like that what you had in mind?

**Comment by daviddias** [comment]
Hi Infrastructure WG!

Now that the Roadmap is a live PR on Github, it is a good time to check in on our Roadmap Instruction Manuals and check in to see if the Roadmap contemplates everything we could have wished for and everything we believe it is valuable for the community to have access. Think things like: Is the Vision described? Are the Requirements well understood? We have a good sense of priority and dependencies? 

You can find the Roadmap Instruction Manuals at:
- https://github.com/ipfs/team-mgmt/blob/master/ROADMAP_HOW_TO.md
- https://github.com/ipfs/team-mgmt/blob/master/TEAM_COORDINATION_TOOLKIT.md#types-of-roadmaps

Check in with your WG and have fun giving this review! :)

File: WG_INFRASTRUCTURE.md (hunk 1 above)
Comment (momack2) [reply WG_INFRASTRUCTURE.md:15]: Something with the same goal, but more specific (ex - "configure and support CI tools for [xyz working groups]", "automate 5 repetitive devops tasks", etc)

**Comment by momack2** [comment]
Merging this PR to document where we are on our roadmaps - future iterations to narrow down on a more focused and achievable set of roadmap initiatives (to match project-level priority update) will follow in next few weeks in a separate PR.

**Pull request merged**